hc/api/tests/test_pause.py

```diff
@@ -25,3 +27,7 @@ def test_it_validates_ownership(self):
         self.assertEqual(r.status_code, 400)
 
         ### Test that it only allows post requests
+        response = self.client.get(url, "", content_type="application/json",
+                             HTTP_X_API_KEY="abc")
+
+        self.assertEqual(response.status_code, 405)
```

Comment:
maintain consistence in coding style
`assert response.status_code == 200` or ` self.assertEqual(response.status_code, 405)`
stick to one style of assertion